### ISSUE TITLE
Elder titan 7.34 astral echo stomp bug

### ISSUE THREAD

**Issue by LoHa123** [ValveSoftware/Dota2-Gameplay#10584]
### Ability name

astral spirit with ecoh stomp

### Description

the ability of the Elder Titan character echo stomp in astral is broken, if you use it while the astral is flying back to the hero, then it will not stop like before, but will simply fly further and will cast the ability and fly to the hero

### Example Match ID (and possibly Timestamp)

7278615880 . 30:05

**Comment by lxvevery1** [comment]
yes, when applying return astral spirit ability and then applying echo stomp ability the spirit would stop and apply this ability correctly. Due to the current bug astral spirit does not stop while echo stomp spell is casting

**Comment by eboug92** [comment]
Fixed now: https://store.steampowered.com/news/app/570/view/7160213079732689910?l=english

Fixed an issue where Elder Titan Astral Spirit would keep flying without stopping when casting Echo Stomp
